### PR TITLE
Sdst 39

### PR DESCRIPTION
**Pull request by VityaPodill**

### CODE REVIEW

File: src/test/java/pages/HomePage.java [hunk 1]
```diff
@@ -44,4 +59,4 @@ public HomePage clickBuildYourOwnButton() {
         waitUntilElementToBeClickableByXpath(BUILD_YOUR_OWN_BUTTON).click();
         return this;
     }
-}
+}
```
Comment (SkipperV) [inline src/test/java/pages/HomePage.java:62]: Add empty space at the end of file

File: src/test/java/pages/BuildYourOwnPage.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        waitUntilElementToBeClickableByXpath(X5_M_SPORTS_SELECT).click();
+        return this;
+    }
+
+    public BuildYourOwnPage selectX5MCompetition() {
+        waitUntilElementToBeClickableByXpath(X5_M_COMPETITION).click();
+        return this;
+    }
+
+    public String checkName() {
```
Comment (Bogdan27nazaruk) [inline src/test/java/pages/BuildYourOwnPage.java:96]: What you check in this method???

File: src/test/java/pages/BuildYourOwnPage.java [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+        waitUntilElementToBeClickableByXpath(NEXT_OPTIONS_BTN).click();
+        return this;
+    }
+
+    public BuildYourOwnPage pressNextSummaryBtn() {
+        waitUntilElementToBeClickableByXpath(NEXT_SUMMARY_BTN).click();
+        return this;
+    }
+
+    public BuildYourOwnPage checkTotalPrice() {
```
Comment (Bogdan27nazaruk) [inline src/test/java/pages/BuildYourOwnPage.java:76]: What you check in this method???

File: src/test/java/pages/BuildYourOwnPage.java [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+    public BuildYourOwnPage selectX5MCompetition() {
+        waitUntilElementToBeClickableByXpath(X5_M_COMPETITION).click();
+        return this;
+    }
+
+    public String checkName() {
+        return waitUntilElementToBeClickableByXpath(CHECK_NAME).getText();
+    }
+
+    public String checkPrise() {
```
Comment (Bogdan27nazaruk) [inline src/test/java/pages/BuildYourOwnPage.java:100]: What you check in this method???

File: src/test/java/tests/CheckModelsField.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+import pages.BuildYourOwnPage;
+import pages.HomePage;
+
+import static org.testng.Assert.assertTrue;
+
+public class CheckModelsField extends AbstractBaseTest {
+
+    @Test
+    public void checkModelsField() {
+
```
Comment (Bogdan27nazaruk) [inline src/test/java/tests/CheckModelsField.java:14]: Please delete empty space!

File: src/test/java/tests/CheckModelsField.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        HomePage homePage = new HomePage(driver);
+        BuildYourOwnPage buildYourOwnPage = new BuildYourOwnPage(driver);
+
+        homePage.clickDismissBtn();
+        homePage.clickModelsBtn();
```
Comment (Bogdan27nazaruk) [inline src/test/java/tests/CheckModelsField.java:19]: You can select next method with dot without ";"

File: src/test/java/tests/CheckModelsField.java [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+        buildYourOwnPage
+                .choosingXM();
+        buildYourOwnPage
```
Comment (Bogdan27nazaruk) [inline src/test/java/tests/CheckModelsField.java:23]: You can select next method with dot without ";"

File: src/test/java/tests/CheckModelsField.java [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+                .pressXMBuildOwn()
+                .selectXMBuild();
+        buildYourOwnPage
```
Comment (Bogdan27nazaruk) [inline src/test/java/tests/CheckModelsField.java:26]: You can select next method with dot without ";"

File: src/test/java/tests/CheckModelsField.java [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+                .closeModalWindow()
+                .pressNextWheelsBtn()
+                .pressNextUpholstery()
+                .selectXMOrangeLeather()
+                .pressNextTrimBtn()
+                .pressNextOptionsBtn()
+                .pressNextAccessories()
+                .pressNextSummaryBtn();
```
Comment (Bogdan27nazaruk) [inline src/test/java/tests/CheckModelsField.java:34]: You can select next method with dot without ";"

File: src/test/java/tests/BuildYourOwnTestX5.java [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+import static org.testng.Assert.assertTrue;
+public class BuildYourOwnTestX5 extends AbstractBaseTest {
+
+    @Test
+    public void checkCustomizeX5M() {
+        BuildYourOwnPage buildYourOwnPage = new BuildYourOwnPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        homePage.clickDismissBtn();
+        homePage.clickBuildYourOwnBtn();
```
Comment (Bogdan27nazaruk) [inline src/test/java/tests/BuildYourOwnTestX5.java:17]: You can select next method with dot without ";"

File: src/test/java/pages/HomePage.java [hunk 11]
```diff
@@ -1,20 +1,20 @@
...
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import elements.HomePageElements;
 
 import static base.CommonActions.jsClick;
 import static base.CommonActions.scrollToElement;
 
 public class HomePage extends HomePageElements {
 
+    private static final String CARS_IN_AVAILABILITY = "//a[@id='button-f4e1544f40'] ";
```
Comment (Bogdan27nazaruk) [inline src/test/java/pages/HomePage.java:12]: It should be in elements page!!

File: src/test/java/tests/CheckModelsFieldTest.java [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+import pages.BuildYourOwnPage;
+import pages.HomePage;
+
+import static org.testng.Assert.assertTrue;
+
+public class CheckModelsFieldTest extends AbstractBaseTest {
+
+    @Test
+    public void checkModelsField() {
+
```
Comment (Bogdan27nazaruk) [inline src/test/java/tests/CheckModelsFieldTest.java:14]: Please delete empty row!!!

**Review by Bogdan27nazaruk — APPROVED** [review]

**Review by TetianaUp — APPROVED** [review]

**Pull request merged**